cloudinit/cmd/devel/__init__.py

```diff
@@ -17,9 +18,17 @@ def addLogHandlerCLI(logger, log_level):
     return logger
 
 
-def read_cfg_paths() -> Paths:
-    """Return a Paths object based on the system configuration on disk."""
+def read_cfg_paths(fetch_existing_datasource: Optional[str] = "") -> Paths:
```

Comment:
Why accept None? If we remove the Optional, don't forget to remove the import too.

Reply:
+1 no need at all to allow None, just default to empty string if unspecified any callers expecting to need get_ipath will have to provide a valid `fetch_existing_datasource=("trust"|"check")`